src/components/ui/form.tsx

```
@@ -47,11 +47,11 @@ const useFormField = () => {
   const { getFieldState, formState } = useFormContext()
 
   if (!fieldContext?.name) {
-    throw new Error("useFormField deve ser usado dentro de <FormField>")
+    throw new Error('useFormField deve ser usado dentro de <FormField>')
   }
   const fieldState = getFieldState(fieldContext.name, formState)
 
-  const { id } = itemContext ?? { id: React.useId() } 
+  const { id } = itemContext ?? { id: React.useId() }
```

ESLint

[error] React Hook "React.useId" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)